src/data/userData.tsx

```diff
@@ -9,6 +9,15 @@ export const Users: User[] = [
     website: "https://my-youtube-ajay.vercel.app/",
     source: "https://github.com/Ajay-Dhangar/youtube_clone",
     tags: ["opensource", "clone", "favorite", "react"],
+  },
+    {
+    title: "Spotify Clone",
+    description:
+      "Build and Deploy a Modern Spotify Clone Application in React JS with Material UI 5",
+    preview: require("./showcase/spotify.png"),
+    website: "https://spotifyclone234.freewebhostmost.com/",
```

Comment:
The current domain (`https://spotifyclone234.freewebhostmost.com/`) is not secure. Could you please replace it with a more secure URL or domain? May be used `github-page`, `vercel`, and some more secure and trustable 

![image](https://github.com/CodeHarborHub/codeharborhub.github.io/assets/99037494/b8a34d48-7444-4265-92ca-09078d89cd94)